src/index.d.ts

```diff
@@ -1,78 +1,82 @@
+import CardImgOverlay from './CardImgOverlay.svelte';
+import CardLink from './CardLink.svelte';
+import CardSubtitle from './CardSubtitle.svelte';
+import CardText from './CardText.svelte';
+import CardTitle from './CardTitle.svelte';
+import Carousel from './Carousel.svelte';
+import CarouselControl from './CarouselControl.svelte';
+import CarouselIndicators from './CarouselIndicators.svelte';
+import CarouselItem from './CarouselItem.svelte';
+import CarouselCaption from './CarouselCaption.svelte';
+import Col from './Col.svelte';
+import Collapse from './Collapse.svelte';
+import Container from './Container.svelte';
+import CustomInput from './CustomInput.svelte';
+import Dropdown from './Dropdown.svelte';
+import DropdownItem from './DropdownItem.svelte';
+import DropdownMenu from './DropdownMenu.svelte';
+import DropdownToggle from './DropdownToggle.svelte';
+import Fade from './Fade.svelte';
+import Form from './Form.svelte';
+import FormFeedback from './FormFeedback.svelte';
+import FormGroup from './FormGroup.svelte';
+import FormText from './FormText.svelte';
+import Icon from './Icon.svelte';
+import Input from './Input.svelte';
+import InputGroup from './InputGroup.svelte';
+import InputGroupAddon from './InputGroupAddon.svelte';
+import InputGroupButtonDropdown from './InputGroupButtonDropdown.svelte';
+import InputGroupText from './InputGroupText.svelte';
+import Jumbotron from './Jumbotron.svelte';
+import Label from './Label.svelte';
+import ListGroup from './ListGroup.svelte';
+import ListGroupItem from './ListGroupItem.svelte';
+import Media from './Media.svelte';
+import Modal from './Modal.svelte';
+import ModalBody from './ModalBody.svelte';
+import ModalFooter from './ModalFooter.svelte';
+import ModalHeader from './ModalHeader.svelte';
+import Nav from './Nav.svelte';
+import Navbar from './Navbar.svelte';
+import NavItem from './NavItem.svelte';
+import NavLink from './NavLink.svelte';
+import NavbarBrand from './NavbarBrand.svelte';
+import NavbarToggler from './NavbarToggler.svelte';
+import Pagination from './Pagination.svelte';
+import PaginationItem from './PaginationItem.svelte';
+import PaginationLink from './PaginationLink.svelte';
+import Popover from './Popover.svelte';
+import Progress from './Progress.svelte';
+import Row from './Row.svelte';
+import Spinner from './Spinner.svelte';
+import Table from './Table.svelte';
+import TabContent from './TabContent.svelte';
+import TabPane from './TabPane.svelte';
+import Toast from './Toast.svelte';
+import ToastBody from './ToastBody.svelte';
+import ToastHeader from './ToastHeader.svelte';
+import Tooltip from './Tooltip.svelte';
+import UncontrolledAlert from './UncontrolledAlert.svelte';
+import UncontrolledButtonDropdown from './UncontrolledButtonDropdown.svelte';
+import UncontrolledCollapse from './UncontrolledCollapse.svelte';
+import UncontrolledFade from './UncontrolledFade.svelte';
+import UncontrolledDropdown from './UncontrolledDropdown.svelte';
```

Comment:
Is this switch to adding `.svelte` correct?

My understanding is this is importing the `*.d.ts` definitions for each compnent and seems to behave correctly in build & editor.